src/components/LeftSidebar/NewGroupConversation/SetConversationType/SetConversationType.vue

```diff
@@ -0,0 +1,77 @@
+<!--
+  - @copyright Copyright (c) 2019 Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @author Marco Ambrosini <marcoambrosini@pm.me>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<div class="conversation-type">
+		<input
+			type="checkbox"
+			:checked="value"
+			@change="handleInput" />
+		<label class="conversation-type__label">{{ t('spreed', 'Make ') }}<strong>{{ conversationDisplayName }}</strong>{{ t('spreed', ' public') }}</label>
+		<p class="conversation-type__hint">
+			{{ t('spreed', `If checked, you will get a link that will allow you to share this conversation with unregistered users once the conversation is created.`) }}
+		</p>
+	</div>
+</template>
+
+<script>
+
+export default {
+	name: 'SetConversationType',
+	props: {
+		value: {
+			type: Boolean,
+			required: true
+		},
+		conversationName: {
+			type: String,
+			default: t('spreed', 'your conversation'),
+		},
+	},
+	computed: {
+		conversationDisplayName() {
+			if (this.conversationName === '') {
+				return t('spreed', 'your conversation')
```

Comment:
```suggestion
				return t('spreed', 'Your conversation')
```

Reply:
Nope, that is a placeholder inline text

Reply:
I knew it ;) , and it was an indirect hint that this will not work with translations in all languages. If it is not a set name, dont use a it inline, but use a full string